### PR TITLE
Compatible with Qt5

### PR DESCRIPTION
**Pull request by anasarrak**

### CODE REVIEW

**Comment by wxmerkt** [comment]
Do the rosdep keys for qt4 resolve to qt5 though? In the package.xml, you only specify qt4, but then have a switch for qt5 in the CMake.

**Comment by anasarrak** [comment]
You are right, ill fix it now

**Comment by anasarrak** [comment]
I think that only the changes made https://github.com/OctoMap/octomap/pull/238/commits/c1a1157e009b124f894e247a1efe982d35bbf64b are not enough, @wxmerkt can you please suggest some advice here?

File: octovis/package.xml [hunk 1]
```diff
@@ -19,6 +19,8 @@
   <depend>libqt4-dev</depend>
   <depend>libqt4-opengl-dev</depend>
 
+  <depend>qtbase5-dev</depend>
+
```
Comment (wxmerkt) [inline octovis/package.xml]: You need to add `libqt5-core`

Comment (anasarrak) [reply octovis/package.xml]: Do I need to replace it or add it as a new depend?

Comment (wxmerkt) [reply octovis/package.xml]: I think add as another, but I haven't tested or looked into it deeply - just checked the correct rosdep key.

**Comment by ahornung** [comment]
In terms of general Qt5 compability (so this is not only limited to the ROS use case), I would suggest separating the changes and targeting the devel branch. The goal should be a working "plain" Qt5 build (not ROS2 package) as first step. Please also have a look at the .travis.yml config for the CI. Before merging, the CI checks must succeed.

**Comment by ahornung** [comment]
In general, Qt5 compatibility was already merged here https://github.com/OctoMap/octomap/pull/152, so maybe this is just a case of adjust the Travis config (for the non-ROS2 use case)?

**Pull request closed without merging**